src/Mod/TechDraw/Gui/QGIViewBalloon.cpp

```diff
@@ -525,22 +540,22 @@ void QGIViewBalloon::placeBalloon(QPointF pos)
 //    Base::Console().Message("QGIVB::placeBalloon(%s)\n",
 //                            DrawUtil::formatVector(pos).c_str());
     auto balloon( dynamic_cast<TechDraw::DrawViewBalloon*>(getViewObject()) );
-    if( balloon == nullptr ) {
+    if (!balloon) {
         return;
     }
 
     DrawView* balloonParent = dynamic_cast<DrawView*>(balloon->SourceView.getValue());
```

Comment:
maybe change this `DrawView*` to `auto`, to make it conform to the style of `balloon`, `featPage `, `vp ` while you're at i

Reply:
dito.